server/routes/probationPractitionerReferrals/ppDashboardPresenter.ts

```diff
@@ -7,13 +7,15 @@ import { SortableTableHeaders, SortableTableRow } from '../../utils/viewUtils'
 import PrimaryNavBarPresenter from '../shared/primaryNavBar/primaryNavBarPresenter'
 import LoggedInUser from '../../models/loggedInUser'
 
-export default class MyCasesPresenter {
+export default class PPDashboardPresenter {
```

Comment:
I tend to shy away from abbreviations (even well-known ones like `PP`) in class names as much as possible for readability. Bearing in mind this is in a directory called `ProbationPractitionerReferrals`, do you think we could just call it `DashboardPresenter` like we do for the SP view, and consider renaming in future if we move it out of this directory?